src/extension.ts

```diff
@@ -169,8 +169,6 @@ export async function attach(
     }
 
     const telemetryProps = { viaConfig: `${!!config}`, withTargetUrl: `${!!attachUrl}` };
-    telemetryReporter.sendTelemetryEvent('command/attach', telemetryProps);
```

Comment:
This might still be useful if the user has already launched the browser (by themselves) and attach to it. IIRC Launch telemetry only covers when we launch the browser.

Reply:
Could you double check to see if this fires upon attach to an existing target? It doesn't seem to fire for me.  Currently, the only extension-side telemetry we fire for attaching to an existing target are the button click for the attach button and the websocket connection.

Reply:
yep you are right not firing either :), so we are good to go.